labs/topologies/lab00_basic_cisco/README.md

```diff
@@ -0,0 +1,348 @@
+## Basic Scenario. Cisco
+
+### Introduction
+
+In this lab, you will gain hands-on experience in deploying simple network configuration across a network infrastructure. This lab is designed to simulate a real-world scenario:
+
+- MTU configuration
+- Description configuration
+
+Author:
+
+- [Vadim Volovik](https://github.com/vadvolo)
+
+### Objectives
+
+- Understand fundamental concepts of Annet
+
+
+### Topology
+
+![Lab Topology](./images/topology.png)
+
+### Environment
+
+- Netbox url: http://localhost:8000/
+- Netbox login/password: `annet/annet`
+- Device telnet and ssh login/password: `annet/annet`
+- Device mgmt addresses:
+   | Router | MGMT |
+   |:------:|:----|
+   | lab-r1 | `172.20.0.111` |
+   | lab-r2 | `172.20.0.112` |
+   | lab-r3 | `172.20.0.113` |
+
+### Preparation
+
+1. Before you start, please put Cisco IOS image `c7200-jk9s-mz.124-13a.bin` into `lab/vm_images` directory.
+   The image is subject to a license agreement, so it cannot be distributed in the repository.
+
+2. Install Docker and Docker Compose on your device
+   - [Docker](https://docs.docker.com/engine/install/)
+     - [Linux](https://docs.docker.com/desktop/install/linux/)
+     - [Mac](https://docs.docker.com/desktop/install/mac-install/)
+     - [Windows](https://docs.docker.com/desktop/install/windows-install/)
+
+3. Install `make` utility:
+   ```bash
+   sudo apt install make  # Linux
+   brew install make      # MacOS
+   ```
+
+### Generators
+
+In this lab, generators are organized within the `./src/lab_generators` directory. The lab utilizes two specific generators:
+
+- Description generator  
+  In this generator, we employ a description pattern for device neighbors formatted as `to_<NEIGHBOR_NAME>_<NEIGHBOR_PORT>`. The device connection map is located in Netbox and is utilized by Annet.
```

Comment:
Too convoluted sentence. Something like 'Generate description like this: `to_<NEIGHBOR_NAME>_<NEIGHBOR_PORT>`' will do